desktop/src/main/java/bisq/desktop/primary/main/content/social/SocialView.java

```diff
@@ -17,30 +17,50 @@
 
 package bisq.desktop.primary.main.content.social;
 
-import bisq.desktop.common.view.NavigationTarget;
-import bisq.desktop.common.view.TabView;
+import bisq.desktop.common.view.*;
+import bisq.desktop.overlay.OverlayWindow;
+import bisq.desktop.primary.main.content.social.gettingStarted.GettingStartedView;
 import bisq.i18n.Res;
+import javafx.beans.value.ChangeListener;
+import javafx.scene.Parent;
+import javafx.scene.layout.Region;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class SocialView extends TabView<SocialModel, SocialController> {
-
+    protected final ChangeListener<View<? extends Parent, ? extends Model, ? extends Controller>> viewChangeListener;
+    
     public SocialView(SocialModel model, SocialController controller) {
         super(model, controller);
 
-        headlineLabel.setText(Res.get("welcome"));
-
         addTab(Res.get("social.gettingStarted"), NavigationTarget.GETTING_STARTED);
         addTab(Res.get("social.discuss"), NavigationTarget.DISCUSS);
         addTab(Res.get("social.learn"), NavigationTarget.LEARN);
         addTab(Res.get("social.connect"), NavigationTarget.CONNECT);
+
+        setupHeadline(model.getView()); // TODO: check why view is null at the beginning
```

Comment:
At onViewAttached it would not be null. But better use EasyBind.subscribe() instead of listeners as that will call the callback at subscribe time, thus not requiring the extra setupHeadline(model.getView()) call.

Reply:
[Fixed](https://github.com/bisq-network/bisq2/pull/296/commits/ef575be7cd4503c8abf8cfc7adac29d1a3e3577f)